portal/src/main/webapp/js/src/study-view/data/StudyViewProxy.js

```diff
@@ -460,6 +573,8 @@ var StudyViewProxy = (function() {
         getArrDataBySampleIds: getArrDataBySampleIds,
         getAttrData: function(){ return obtainDataObject.attr;},
         getMutatedGenesData: getMutatedGenesData,
+        getMutatedGeneDataBasedOnSampleIds: getMutatedGeneDataBasedOnSampleIds,
```

Comment:
I certainly agree. @zhx828, are we right in assuming that getMutatedGeneDataBasedOnSampleIds is doing the same thing as QuerySession.getGenomicEventData({genetic_profile_ids: [..], genes:[..], sample_ids:[..]}) ?